tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/JavaLanguageSpecificChecks.cs

```diff
@@ -257,8 +248,24 @@ public async Task<CLICheckResponse> LintCodeAsync(string packagePath, bool fixCh
         }
         results.Add(("RevAPI", revapiHasIssues));
 
-        _logger.LogInformation("Linting results parsed: Checkstyle ran={CheckstyleRan} issues={CheckstyleIssues}, SpotBugs ran={SpotBugsRan} issues={SpotBugsIssues}, RevAPI ran={RevapiRan} issues={RevapiIssues}",
-            checkstyleRan, checkstyleHasIssues, spotbugsRan, spotbugsHasIssues, revapiRan, revapiHasIssues);
+        // Check for Javadoc execution and results (included in linting)
+        var javadocRan = output.Contains("maven-javadoc-plugin") && !output.Contains("-Dmaven.javadoc.skip=true");
+        var javadocHasIssues = false;
+        if (javadocRan)
+        {
+            // Look for javadoc errors or warnings
+            var javadocSuccess = !output.Contains("Javadoc Warnings") && 
```

Comment:
should this get added into the ParseLintingResults() fn or is it quite different from the other formats?